docs/05/d.md

```diff
@@ -34,7 +34,7 @@ Here's [the typeclass contract for `MonadFilter`][MonadFilterSource]:
 }
 ```
 
-This enables a derivative function/operator `filter` and `filterM`:
+This enables the derivative functions/operators `filter` and `filterM`:
```

Comment:
The previous felt weird because you spoke of a single function but listed two names.  Now perhaps it's appropriate to think of `filterM` as an alias, but that's not usually how I roll.  
I don't think a comma is appropriate because you're not really in a place for an appositive. You don't really need to signify the kind of expectant break that a colon delivers. 
If anything I think I'd just suggest going to either "function" or "operator" as the difference in Scala is contextual and largely immaterial.